### PR TITLE
Move autocompletion scripts to bin/

### PR DESCRIPTION
**Pull request by zdroid**
Creates a bit less mess in main tree. And autocompletion files are scripts anyway.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ plugins=(git tmux tldr)
 ```
 
 Fret not regular zsh user!
-Copy or symlink `autocompletion/tldr.zsh` to
+Copy or symlink `bin/tldr.zsh` to
```
Comment (waldyrious) [inline README.md]: didn't you forget the "autocompletion" in the new path?

Comment (zdroid) [inline README.md]: It should be fine now.

**Comment by waldyrious** [comment]
LGTM, waiting for @agnivade to comment as he mentioned he was working on a separate PR.

**Comment by agnivade** [comment]
Its fine. This can be pushed. I plan to make some broad level changes later.

**Pull request merged**